pkg/env/prod.go

```diff
@@ -128,7 +128,8 @@ func newProd(ctx context.Context, log *logrus.Entry) (*prod, error) {
 
 	p.serviceKeyvault = keyvault.NewManager(msiKVAuthorizer, serviceKeyvaultURI)
 
-	err = p.populateZones(ctx, msiAuthorizer)
+	resourceSkusClient := compute.NewResourceSkusClient(p.Environment(), p.SubscriptionID(), msiAuthorizer)
+	err = p.populateVMSkus(ctx, resourceSkusClient)
```

Comment:
Any reason this logic changed in the way that:
before: client initiated inside `populateZones` and authorized passed in.
now: client initiated outside an client passed in?

Reply:
@mjudeikis the only reasons is to make `populateZones` (now `populateVMSkus`) more unit testable. I did not add any extra coverage as part of this PR